app/server/api.py

```diff
@@ -209,13 +209,21 @@ def get(self, request, *args, **kwargs):
         project = get_object_or_404(Project, pk=self.kwargs['project_id'])
         documents = project.documents.all()
         painter = self.select_painter(format)
-        data = painter.paint(documents)
+        
+        # json1 format prints text labels while json format prints annotations with label ids
+        # json1 format - "labels": [[0, 15, "PERSON"], ..]
+        # json format - "annotations": [{"label": 5, "start_offset": 0, "end_offset": 2, "user": 1},..]
+        if format == "json1":
```

Comment:
Instead of this conditional, should the `json1` format be a separate painter that extends or inherits from the JSONPainter? This will make the code more single-responsibility and will also make it easier in the future to evolve the two classes separately.

Reply:
I did consider that, but I thought it would be confusing to have two classes for JSON files. In this case my goal was for each function to cover a single responsibility. Also, given that there are other json types that people would like to export (e.g. Spacy) this class would handle all of them but have different functions for each format.  Thoughts?